<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qimvi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/qimvi.qrc">
    <normaloff>:/fugue/fugue-icons/image-sunset.png</normaloff>:/fugue/fugue-icons/image-sunset.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTreeView" name="dirTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="fileListView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="imagePreview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="backgroundBrush">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>207</red>
             <green>207</green>
             <blue>207</blue>
            </color>
           </brush>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWidok">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionFitWindow"/>
    <addaction name="actionFitActual"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuObraz">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionCrop"/>
    <addaction name="actionResize"/>
    <addaction name="separator"/>
    <addaction name="actionRotateRight"/>
    <addaction name="actionRotateLeft"/>
    <addaction name="separator"/>
    <addaction name="actionToGrayscale"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdycja"/>
   <addaction name="menuWidok"/>
   <addaction name="menuObraz"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionFitWindow"/>
   <addaction name="actionFitActual"/>
   <addaction name="separator"/>
   <addaction name="actionRotateRight"/>
   <addaction name="actionRotateLeft"/>
   <addaction name="separator"/>
   <addaction name="actionCrop"/>
   <addaction name="actionResize"/>
   <addaction name="separator"/>
   <addaction name="actionToGrayscale"/>
  </widget>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/disk--pencil.png</normaloff>:/fugue/fugue-icons/disk--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/door.png</normaloff>:/fugue/fugue-icons/door.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/layer-resize.png</normaloff>:/fugue/fugue-icons/layer-resize.png</iconset>
   </property>
   <property name="text">
    <string>Fit preview in window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionFitActual">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/layer-resize-actual.png</normaloff>:/fugue/fugue-icons/layer-resize-actual.png</iconset>
   </property>
   <property name="text">
    <string>Actual size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/magnifier-zoom-in.png</normaloff>:/fugue/fugue-icons/magnifier-zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/magnifier-zoom-out.png</normaloff>:/fugue/fugue-icons/magnifier-zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/disk.png</normaloff>:/fugue/fugue-icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/layer-rotate.png</normaloff>:/fugue/fugue-icons/layer-rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 90째 right</string>
   </property>
   <property name="toolTip">
    <string>Obr처t o 90째 w prawo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/layer-rotate-left.png</normaloff>:/fugue/fugue-icons/layer-rotate-left.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 90째 left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/arrow-curve-180-left.png</normaloff>:/fugue/fugue-icons/arrow-curve-180-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/arrow-curve.png</normaloff>:/fugue/fugue-icons/arrow-curve.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/information.png</normaloff>:/fugue/fugue-icons/information.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/image-crop.png</normaloff>:/fugue/fugue-icons/image-crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop to selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/image-resize.png</normaloff>:/fugue/fugue-icons/image-resize.png</iconset>
   </property>
   <property name="text">
    <string>Resize...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/application-resize-full.png</normaloff>:/fugue/fugue-icons/application-resize-full.png</iconset>
   </property>
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionToGrayscale">
   <property name="icon">
    <iconset resource="../res/qimvi.qrc">
     <normaloff>:/fugue/fugue-icons/color_gray.png</normaloff>:/fugue/fugue-icons/color_gray.png</iconset>
   </property>
   <property name="text">
    <string>Convert to greyscale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/qimvi.qrc"/>
 </resources>
 <connections/>
</ui>
